docs/CONFIG-VARS.md

```diff
@@ -190,7 +190,7 @@ Ubuntu 20.04 LTS is the operating system used on the Jump/NFS servers. Ubuntu cr
 
 ### Additional Node Pools
 
-Additional node pools can be created separate from the default node pool. This is done with the `node_pools` variable, which is a map of objects. Each node pool requires the following variables:
+Additional node pools can be created separate from the default node pool. This is done with the `node_pools` variable, which is a map of objects. Irrespective of the default values, the following variables are required for each node pool:
```

Comment:
The old wording is just fine. As it states `Each node pool requires` What's the confusion here? The defaults are only used if you have not defined ANY node pools yourself.